dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/AbstractConfiguratorListener.java

```diff
@@ -103,21 +103,27 @@ public void process(ConfigChangedEvent event) {
     }
 
     private boolean genConfiguratorsFromRawRule(String rawConfig) {
-        boolean parseSuccess = true;
+        List<URL> urls;
         try {
             // parseConfigurators will recognize app/service config automatically.
-            List<URL> urls = ConfigParser.parseConfigurators(rawConfig);
-            List<URL> safeUrls = urls.stream()
-                .map(url -> url.removeParameters(securityKey))
-                .map(url -> url.setScopeModel(moduleModel))
-                .collect(Collectors.toList());
-            configurators = Configurator.toConfigurators(safeUrls).orElse(configurators);
+            urls = ConfigParser.parseConfigurators(rawConfig);
         } catch (Exception e) {
-            logger.error("Failed to parse raw dynamic config and it will not take effect, the raw config is: " +
-                    rawConfig, e);
-            parseSuccess = false;
+            // support single ip (issue: #8821 #9239)
+            URL url = URL.valueOf(rawConfig);
```

Comment:
Why need continue  parse Illegal rawConfig?

Reply:
org.apache.dubbo.remoting.zookeeper.curator#createPersistent(String path) and createEphemeral(String path) will call
org.apache.curator.framework.imps.CreateBuilderImpl#forPath(path) and transfer client.getDefaultData() - client ip address - as dataChanged event content to the provider's org.apache.dubbo.remoting.zookeeper.curator.CuratorZookeeperClient#NodeCacheListenerImpl#nodeChanged. 
if client ip address is not acceptable, it should be filtered out of the event process instead of being processed by AbstractConfiguratorListener.

Reply:
It doesn't make sense to accept a single IP string as a URL.

Reply:
ok, i'll delete it.
filtering out single ip address is more complex than i expected, see regex pattern of ipv6:
```
(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))
```
https://ihateregex.io/expr/ipv6/

Reply:
Maybe there is illegal input by the user,I think there is no problem with illegal input throwing an exception.